packages/react-notion-x/src/styles.css

```diff
@@ -1600,6 +1600,323 @@ svg.notion-page-icon {
   text-overflow: ellipsis;
 }
 
+.notion-calendar-view {
+  position: relative;
+  padding-left: 1px;
+}
+
+.notion-calendar-header {
+  position: absolute;
+  left: 0px;
+  right: 0px;
```

Comment:
quick tip; for css lengths `0` is a special value where you don't need to specify the unit. so `0px` is the same as `0`. just a nit & not important, but something I try to follow